### PR TITLE
Add quimb docset

### PR DESCRIPTION
**Pull request by mofeing**
Here I open a PR to add the docset for [`quimb`](https://github.com/jcmgray/quimb) a Python package for many-body physics using tensor networks. It is increasingly used by researchers in quantum _things_.

### CODE REVIEW

File: docsets/quimb/docset.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
+    "name": "quimb",
+    "version": "1.3.0",
+    "archive": "quimb.tgz",
```
Comment (Kapeli) [inline docsets/quimb/docset.json:4]: You need to add this archive as well.

**Comment by Kapeli** [comment]
`quimb.tgz` needs to contain just the docset. Currently the archive contains a `_build` folder and the docset is located within that folder.

Archive the docset using:

```
tar --exclude='.DS_Store' -cvzf <docset name>.tgz <docset name>.docset
```

**Comment by mofeing** [comment]
ups 😅. solved!

**Comment by Kapeli** [comment]
Please note that you need to add the main archive as well (see comment [here](https://github.com/Kapeli/Dash-User-Contributions/pull/3580#discussion_r805162901)). I've added it this time by copying the versioned archive.

**Comment by Kapeli** [comment]
Thank you! 👍

**Comment by mofeing** [comment]
> Please note that you need to add the main archive as well (see comment [here](https://github.com/Kapeli/Dash-User-Contributions/pull/3580#discussion_r805162901)). I've added it this time by copying the versioned archive.

Sorry, I don't understand. For the next time, do you mean I have to include the JSON inside the archived file?

**Comment by Kapeli** [comment]
No. You need to include this file: `docsets/quimb/quimb.tgz` (this is the archive for the main version of the docset) and also `docsets/quimb/versions/1.3.0/quimb.tgz` (the archive for the specific version of the docset).

**Pull request merged**